README

```diff
@@ -1,4 +1,4 @@
-pilot-link -- Palm Handheld Glue - http://www.pilot-link.org
+pilot-link -- Palm Handheld Glue - https://sourceforge.net/projects/pilot-link/
```

Comment:
Sourceforge has NEVER been the home of pilot-link. Let's correct this to point back to the correct TLD for the project and resubmit an updated PR.